cli/tests/pcluster/config/test_update_policy.py

```diff
@@ -807,3 +814,415 @@ def test_shared_storage_update_policy_condition_checker(
+                                    "Networking": {"PlacementGroup": {"Enabled": True}},
+                                }
+                            ],
+                            "Networking": {"PlacementGroup": {"Enabled": True}},
+                        }
+                    ]
+                }
+            },
+            {
+                "Scheduling": {
+                    "Queues": [
+                        {
+                            "Name": "mock-q",
+                            "ComputeResources": [
+                                {
+                                    "Name": "cr-pg-enabled",
+                                    "Networking": {"PlacementGroup": {"Enabled": False}},
+                                }
+                            ],
+                            "Networking": {"PlacementGroup": {"Enabled": True}},
+                        }
+                    ]
+                }
+            },
+            Change(
+                path=["Queues[mock-q]", "ComputeResources[cr-pg-enabled]"],
+                key="",
+                old_value="",
+                new_value="",
+                update_policy={},
+                is_list=False,
+            ),
+            True,
+            True,
+            False,
+            "All compute nodes must be stopped for a managed placement group deletion",
+        ),
+        # Negative test case, disable at the cr level, while enabled at q
+        pytest.param(
+            {
+                "Scheduling": {
+                    "Queues": [
+                        {
+                            "Name": "mock-q",
+                            "ComputeResources": [
+                                {
+                                    "Name": "cr-pg-enabled",
+                                    "Networking": {"PlacementGroup": {"Enabled": True}},
+                                }
+                            ],
+                        }
+                    ]
+                }
+            },
+            {
+                "Scheduling": {
+                    "Queues": [
+                        {
+                            "Name": "mock-q",
+                            "ComputeResources": [
+                                {
+                                    "Name": "cr-pg-enabled",
+                                }
+                            ],
+                        }
+                    ]
+                }
+            },
+            Change(
+                path=["Queues[mock-q]", "ComputeResources[cr-pg-enabled]"],
+                key="",
+                old_value="",
+                new_value="",
+                update_policy={},
+                is_list=False,
+            ),
+            True,
+            True,
+            False,
+            "All compute nodes must be stopped for a managed placement group deletion",
+        ),
+    ],
+)
+def test_condition_checker_managed_placement_group(
```

Comment:
Can we test a scenario where the `base_config` has more than one queue, and the `target_config` includes `PlacementGroup` changes in two or more of the queues?

Reply:
I can add a unit test to tackle this scenario, but I'd like to avoid the explosion of possibilities when adding more queues and more changes.  What is the main function you want to see tested here?  The queue and compute resource selection process?

Reply:
> The queue and compute resource selection process?
Exactly 👍.

Reply:
sounds good!  Added!